.github/workflows/spike-openocd-tests.yml

```diff
@@ -0,0 +1,76 @@
+# Build spike and run a couple of debug tests.
+
+name: Test OpenOCD against 2 spike configurations
+
+on: pull_request
+
+# There is some commented out code below that would be useful in adding this
+# workflow to other repos. Ideally we can come up with something that would
+# leave this file almost identical between repos, so they can all easily run
+# this test suite.
```

Comment:
I thought that other forks of riscv-openocd could use the workflow file as is - without changes. What would be the commented pieces of code for?

Reply:
To deal with source showing up in different directories, and also to optionally check out a specific git commit instead of just the latest. Compare this PR with https://github.com/riscv-software-src/riscv-isa-sim/blob/master/.github/workflows/debug-smoke.yml

Reply:
My workflow has always attempted to ensure that mainline spike, OpenOCD and riscv-tests always pass. Generally the process is:
1. Add feature to spike.
2. Add feature to OpenOCD.
3. Add test to riscv-tests

I would rather know that we're always testing the latest stuff and sometimes deal with a problem resulting from that, than not be testing the latest stuff (and frequently forgetting/having to update the hashes in the workflow).

Reply:
Ok, understood, thank you for explanation.